src/main/java/io/kuzzle/sdk/core/Collection.java

```diff
@@ -223,6 +223,135 @@ public void onError(JSONObject error) {
+    this.scrollSpecifications(scrollId, new Options(), listener);
+  }
+
+  /**
+   * Scrolls through specifications using the provided scrollId
+   *
+   * @param scrollId string
+   * @param options Options Optional parameters
+   * @param listener Response callback
+   */
+  public void scrollSpecifications(@NonNull final String scrollId, final Options options, @NonNull final ResponseListener<JSONObject> listener) {
+    this.kuzzle.isValid();
+
+    JSONObject data = new JSONObject();
+
+    if (scrollId == null) {
+      throw new RuntimeException("Collection.scrollSpecifications: scrollId is required");
+    }
+
+    if (listener == null) {
+      throw new IllegalArgumentException("listener cannot be null");
+    }
+
+    try {
+      data.put("scrollId", scrollId);
+
+      this.kuzzle.addHeaders(data, this.getHeaders());
+
+      this.kuzzle.query(makeQueryArgs("collection", "scrollSpecifications"), data, options, new OnQueryDoneListener() {
+        @Override
+        public void onSuccess(JSONObject response) {
+          try {
+            listener.onSuccess(response.getJSONObject("result"));
+          } catch (JSONException e) {
+            throw new RuntimeException(e);
+          }
+        }
+
+        @Override
+          public void onError(JSONObject error) {
+            listener.onError(error);
+          }
+      });
+    } catch (JSONException e) {
+      throw new RuntimeException(e);
+    }
+  }
+
+  /**
+   * Searches specifications across indexes/collections according to the provided filters
+   *
+   * @param listener Response callback
+   */
+  public void searchSpecifications(@NonNull final ResponseListener<JSONObject> listener) {
+    this.searchSpecifications(null, new Options(), listener);
+  }
+
+  /**
+   * Searches specifications across indexes/collections according to the provided filters
+   *
+   * @param filters JSONObject Optional filters in ElasticSearch Query DSL format
+   * @param listener Response callback
+   */
+  public void searchSpecifications(final JSONObject filters, @NonNull final ResponseListener<JSONObject> listener) {
+    this.searchSpecifications(filters, new Options(), listener);
+  }
+
+  /**
+   * Searches specifications across indexes/collections according to the provided filters
+   *
+   * @param options Options Optional parameters
+   * @param listener Response callback
+   */
+  public void searchSpecifications(final Options options, @NonNull final ResponseListener<JSONObject> listener) {
+    this.searchSpecifications(null, options, listener);
+  }
+
+  /**
+   * Searches specifications across indexes/collections according to the provided filters
+   *
+   * @param filters JSONObject Optional filters in ElasticSearch Query DSL format
+   * @param options Options Optional parameters
+   * @param listener Response callback
+   */
+  public void searchSpecifications(final JSONObject filters, final Options options, @NonNull final ResponseListener<JSONObject> listener) {
```

Comment:
Same here: other search methods resolve to a `SearchObject` object, maybe this one should too?